moveit_ros/moveit_servo/CMakeLists.txt

```diff
@@ -99,7 +99,8 @@ rclcpp_components_register_nodes(${SERVO_COMPONENT_NODE} "moveit_servo::ServoNod
 
 # Add executable for using a controller
 add_library(${SERVO_CONTROLLER_INPUT} SHARED src/teleop_demo/joystick_servo_example.cpp)
-ament_target_dependencies(${SERVO_CONTROLLER_INPUT} ${THIS_PACKAGE_INCLUDE_DEPENDS})
+ament_target_dependencies(${SERVO_CONTROLLER_INPUT} PUBLIC ${THIS_PACKAGE_INCLUDE_DEPENDS})
```

Comment:
Why should this be PUBLIC? Nothing else should ever depend on it.

Make the same change at L72, 83, 87, 96, 113, 118, 123, for consistency.

Reply:
I went ahead and pushed this change. Please review! I'm happy with the PR now.

Reply:
I'm sorry I missed your message, I think I tried having it as `PRIVATE` but it failed for humble, which is why I changed them back to Public (the default when nothing is specified?)

Reply:
@AndyZe @JafarAbdi No privacy for ament_target_dependencies :sweat_smile:  https://github.com/ament/ament_cmake/issues/158#issuecomment-475384147

Reply:
& because of this we cannot use private in target_link_libraries either :disappointed: